tests/audio/test_pesq.py

```diff
@@ -0,0 +1,138 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from pesq import pesq as pesq_backend
+from torch import Tensor
+
+from tests.helpers import seed_all
+from tests.helpers.testers import BATCH_SIZE, NUM_BATCHES, MetricTester
+from torchmetrics.audio import PESQ
+from torchmetrics.functional import pesq
+from torchmetrics.utilities.imports import _TORCH_GREATER_EQUAL_1_6
+
+seed_all(42)
+
+Input = namedtuple("Input", ["preds", "target"])
+
+# for 8k sample rate, need at least 8k/4=2000 samples
+inputs_8k = Input(
+    preds=torch.rand(NUM_BATCHES, BATCH_SIZE, 2100),
+    target=torch.rand(NUM_BATCHES, BATCH_SIZE, 2100),
+)
+# for 16k sample rate, need at least 16k/4=4000 samples
+inputs_16k = Input(
+    preds=torch.rand(NUM_BATCHES, BATCH_SIZE, 4100),
+    target=torch.rand(NUM_BATCHES, BATCH_SIZE, 4100),
+)
```

Comment:
these tests take a really long time to run currently, so lets lower the data that needs to be feed through
```suggestion
inputs_8k = Input(
    preds=torch.rand(4, 8, 2100),
    target=torch.rand(4, 8, 2100),
)
# for 16k sample rate, need at least 16k/4=4000 samples
inputs_16k = Input(
    preds=torch.rand(4, 8, 4100),
    target=torch.rand(4, 8, 4100),
)
```